### DISCUSSION TITLE
Release v1.1.0 on 22 December

### DISCUSSION THREAD

**Discussion by Alcoz** [datafoodconsortium/taxonomies#19] [Announcements]
We will be releasing version 1.1.0 on 22 December.

# Added

- vocabulary.rdf
- vocabulary.json
- Chicken in Facets
- ChickenPart in Facets

# Fixed

- Roster URI replace by Rooster URI in Facets

**Comment by RaggedStaff** [comment]
@Alcoz  - Has this release been actioned? I can't see it. :thinking:
